frontend/tailwind.config.ts

```diff
@@ -37,7 +35,7 @@ export default {
       lhListBoxTitle: '47px',
     },
     fontFamily: {
-      sansTitle: ['DM Sans', 'sans-serif'],
+      sansTitle: ["'DM Sans Variable'", 'sans-serif'],
```

Comment:
Are the double quotes intentional?

Reply:
ya, i just added a comment; you need nested quotes to handle font names with spaces in them